guillotina_amqp/api.py

```diff
@@ -1,11 +1,12 @@
+from .exceptions import TaskNotFoundException
+from .state import get_state_manager
+from .state import TaskState
 from guillotina import configure
+from guillotina.interfaces import IApplication
 from guillotina.response import HTTPNotFound
-from .state import TaskState
-from .state import get_state_manager
-from .exceptions import TaskNotFoundException
 
 
-@configure.service(method='GET', name='@amqp-tasks',
+@configure.service(method='GET', name='@amqp-tasks', context=IApplication,
                    permission='guillotina.ManageAMQP',
```

Comment:
shouldn't it be IContainer?

Reply:
I think it's dangerous to do on IContainer.

These tasks are global across multiple containers. If you set it on IContainer, a container owner could self-assign local perms to have access to everyone else's tasks(Yes, I know often you can trust container owners).

Reply:
If we want it maintained on IContainer, we'll have to iterate through them all and only provide the ones for this container.

Or, better yet maybe.. We prefix the task ids with the container id always--that sounds like a good idea...

Reply:
yes I agree! prefixing task ids with container ids its a must, in any case.

not sure about the implications of the endpoints at the IApplication or IContainer level though... aren't plugins installed on a container basis?

Reply:
`applications` can affect the entire guillotina application.

`addons` are for only containers though. This guillotina application is not only for containers though!

Reply:
Changed to filter on a prefix!